src/main/kotlin/world/anhgelus/msmp/alteredrespawn/DefaultCondition.kt

```diff
@@ -0,0 +1,18 @@
+package world.anhgelus.msmp.alteredrespawn
+
+import org.bukkit.GameMode
+import org.bukkit.entity.Player
+
+class DefaultCondition {
+    fun player(player: Player){
+        if(player.isOp){
+            player.sendMessage("§7As administrator you gamemode hasn't been changed")
```

Comment:
Do not use the `player.sendMessage(string)`, use the `ChatHelper` object provided in `MSMPCore`